traffic_portal/app/src/common/modules/table/changeLogs/table.changeLogs.tpl.html

```diff
@@ -20,35 +20,45 @@
 <div class="x_panel">
     <div class="x_title">
         <ol class="breadcrumb pull-left">
-            <li class="active">Change Logs</li>
+            <li class="active">Change Logs <small>[ last <a ng-click="changeDays()">{{days}}</a> days ]</small></li>
```

Comment:
If this isn't a link, should use `button[type="button"]` instead of `a`.

Reply:
yes